### ISSUE TITLE
Apply mobile UI to Fans, Lighting, Wastewater and Pump calculators

### ISSUE THREAD

**Issue by rhernandez-intertech** [ORNL-AMO/AMO-Tools-Desktop#5855]

**Comment by rhernandez-intertech** [comment]
Pumps calcs that need mobile UI : Pump Head Tool.

**Comment by koay9f** [comment]
Going to throw this back also.
Pump & Fan curve's "by data" looks a little wonky
If this is a quick fix, go for it, if not, close this and let's make a new issue for later.
Maybe have the units be up with teh headers? (Flow (gpm), Head (ft), Power (hp))
As is this is really hard to follow
![image.png](https://images.zenhubusercontent.com/5cd48a2af8cffa5a19122d27/1a67b98f-7db0-4375-b112-3c30c99b7809)

**Comment by koay9f** [comment]
Also for the Fan traverse.  
It was almost wonderful. We need to think of another option for the traverse data though.  This might have to be a new issue.

![image.png](https://images.zenhubusercontent.com/5cd48a2af8cffa5a19122d27/399922b7-6049-496e-866d-6c75f4ddf605)
